app/src/main/java/com/nervousfish/nervousfish/modules/pairing/APairingHandler.java

```diff
@@ -0,0 +1,15 @@
+package com.nervousfish.nervousfish.modules.pairing;
+
+/**
+ * Contains common methods shared by all pairing modules.
+ */
+@SuppressWarnings("PMD.AbstractClassWithoutAbstractMethod")
+abstract class APairingHandler {
+
+    /**
+     * Prevent instatiation from non-subclasses outside the package
+     */
+    @SuppressWarnings("PMD.UncommentedEmptyConstructor")
+    protected APairingHandler() {
+    }
```

Comment:
Consider removing the suppression, adding the JavaDoc as a comment inside the method and changing the JavaDoc to "Simple {@code APairingHandler} constructor without parameters."